src/cryptonote_core/blockchain.cpp

```diff
@@ -4004,7 +4005,7 @@ bool verify_network_block(std::vector<std::string> &block_verifiers_database_has
   // check if the blocks reserve bytes hash matches any of the network data nodes
   VERIFY_DATA_HASH(BLOCK_VERIFIERS_TOTAL_AMOUNT,block_verifiers_database_hashes,block_verifier_count);
 
-  if (block_verifier_count >= BLOCK_VERIFIERS_VALID_AMOUNT)
+  if (block_verifier_count >= BLOCK_VERIFIERS_VALID_AMOUNT||(hashesStatus==1&&block_verifier_count>=1))
```

Comment:
Hi @snakewaypasser 
I dont fully understand this

This part is where we need 27+ block verifiers to say "yes the current block you received from some random xcashd (does not have to be in xcash dpops) is the same block we have

But then this code you add, where hashesStatus is 1 and at least 1 block verifier added it. It does not make sense to me because wont this bypass the 27+ DBFT check if just 1 block verifier says yes?

I looked where hashesStatus is set to 1 and its only when passRootRequest>=2
passRootRequest>=2 is run if at least 2 root (DPOPS seed) nodes return a hash response

So to me this would mean if at least 2 root nodes return any response (not necessarily the correct hash) then say yes and add the block to your local chain?